File: blog_springBoot/src/main/java/com/synergism/blog/core/article/entity/ArticleInformation.java
package com.synergism.blog.core.article.entity;


import com.synergism.blog.core.classify.entity.Classify;
import com.synergism.blog.core.tag.entity.Tag;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

/**
 * 文章详细信息
 */
@Getter
@Setter
public class ArticleInformation extends Article {
     //所属者昵称
    private String nickname;
    //所属者账号
    private String username;
    //评论数
    private int commentCount;
    //分类
    private Classify classify;
    //标签
    private List<Tag> tagList;
}

File: blog_vue/src/model/article/ArticleInformation.ts
import Classify from "../classify/Classify";
import Tag from "../tag/Tag";
import TagInformation from "../tag/TagInformation";
import Article from "./Article";

export default class ArticleInformation extends Article {
  /**所属者昵称 */
  nickname: string;
  /**所属者账号 */
  username: string;
  /**评论数 */
  commentCount: number;
  /**分类 */
  classify: Classify;
  /**标签列表 */
  tagList: Array<Tag>;

  constructor() {
    super();
    this.nickname = "";
    this.username = "";
    this.commentCount = -1;
    this.classify = new Classify();
    this.tagList = [new TagInformation()];
  }
}
